src/assets/icons/symbol-driving-active.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="28" height="28" xmlns="http://www.w3.org/2000/svg">
+    <g fill="none" fill-rule="evenodd">
+        <circle fill="#4D4D4D" cx="14" cy="14" r="13"/>
```

Comment:
You could this svg for both `symbol-driving` and `symbol-driving-active` by changing this line in
`<circle fill="currentColor" cx="14" cy="14" r="13"/>` and setting `color: transparent` or `color: #4D4D4D` with CSS. Same goes for the other svgs

Reply:
It just hit me that I need to change two colour for this, graphic colour (dark grey - white)  and the background colour (transparent - dark grey). Is there any way to do this via SVG? 

I can think of a solution where I make the background the hosting element. I would make it a circle using rounded corners. Which is what I'm going to do if you don't have any other suggestions?

Reply:
You can play around with `color`, `background-color` (and `border-radius`) and `fill` get the desired effect

Reply:
I resolved this by ultimately creating an icon component to feed inline svg icons into the templates. This is not as elegant as it could be, but I'll get to that later on in the project.